src/Hal/Serializer/ItemNormalizer.php

```diff
@@ -181,7 +181,7 @@ private function populateRelation(array $data, $object, string $format = null, a
 
             $relationName = $relation['name'];
             if ($this->nameConverter) {
-                $relationName = $this->nameConverter->normalize($relationName);
+                $relationName = $this->nameConverter->normalize($relationName, $class, $format);
```

Comment:
the 4th parameter `$context` is available here, (and probably in other places). why not pass it?

Reply:
I'm really not sure it makes sense, I thought about it but I don't know what informations the context brings here that can be useful.

Reply:
i guess if a user has implemented an `AdvancedNameConverterInterface` which uses the context (groups)?